packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanDetail/AccordianLogs.js

```diff
@@ -57,13 +69,13 @@ export default function AccordianLogs(props: AccordianLogsProps) {
               // eslint-disable-next-line react/no-array-index-key
               key={`${log.timestamp}-${i}`}
               className={i < logs.length - 1 ? 'ub-mb1' : null}
-              // compact
-              highContrast
-              isOpen={openedItems.has(log)}
-              linksGetter={linksGetter}
               data={log.fields || []}
+              highContrast
+              interactive={interactive}
+              isOpen={openedItems ? openedItems.has(log) : false}
               label={`${formatDuration(log.timestamp - timestamp)}`}
-              onToggle={() => onItemToggle(log)}
+              linksGetter={linksGetter}
+              onToggle={interactive && onItemToggle ? () => onItemToggle(log) : null}
```

Comment:
No sure what the preference/recommendation is.

Would it make sense to use undefined instead of null as it is defined as "onToggle?" in the properties?

More of a general question about using null vs undefined than anything.

Reply:
As a practice, I generally use `null`, which differentiates something set to `null` from something that's not there. E.g.

```js
const o = {};
console.log(o.notThere === null); // false
// for convenience, `==` is easier than `value === null || value === undefined`
console.log(o.notThere == null);  // true
```